Merge debian 8.8 branch

trombik opened the pull request:
fixes #21

Commit:
introduce Debian 8.8

Commit:
Merge branch 'debian_8.8' into merge_debian_8.8

Commit:
add local modification

mitsururike commented on debian-8.8-amd64.json:
```diff
@@ -0,0 +1,109 @@
+    "iso_checksum": "{{user `iso_checksum`}}",
+    "iso_checksum_type": "{{user `iso_checksum_type`}}",
+    "output_directory": "output-debian-8.8-amd64-{{build_type}}",
+    "vm_name": "packer-debian-8.8-amd64",
```
`"hard_drive_interface": "scsi",`

mitsururike commented on http/debian-8.8/preseed.cfg:
```diff
@@ -0,0 +1,58 @@
+d-i preseed/late_command string                                                   \
+        echo 'Defaults:vagrant !requiretty' > /target/etc/sudoers.d/vagrant;      \
+        echo 'vagrant ALL=(ALL) NOPASSWD: ALL' >> /target/etc/sudoers.d/vagrant;  \
+        chmod 440 /target/etc/sudoers.d/vagrant
```
`        chmod 440 /target/etc/sudoers.d/vagrant;                                  \`
`        sed -i -e 's/^GRUB_TIMEOUT=.*$/GRUB_TIMEOUT=0/' /target/etc/default/grub; \`
`        in-target update-grub`

Commit:
use SCSI

https://github.com/reallyenglish/packer-templates/pull/24#pullrequestreview-41706637

Commit:
do not wait in GRUB menu

https://github.com/reallyenglish/packer-templates/pull/24#discussion_r119795253

Commit:
update archlinux iso_url

Commit:
another iso_url fix

Commit:
Merge branch 'debian_8.8' into merge_debian_8.8

Commit:
use mirror
